nx/lib/nx/pytorch_backend.ex

```diff
@@ -0,0 +1,177 @@
+defmodule Nx.PytorchBackend do
+  @behaviour Nx.Backend
+
+  defstruct [:ref]
+
+  alias Nx.Pytorch.NIF
+
+  import Nx.Shared
```

Comment:
You are not supposed to use this module once you move this to a separate, because it is private for Nx. I see you have using it for iota though, so I will have a suggestion on how to refactor it.